.github/workflows/linting.yml

```
@@ -36,7 +36,7 @@ jobs:
       - name: Checkout this repository
         uses: actions/checkout@v3
         with:
-          repository: pmagentur/linting-javascript
+          repository: pmagentur/linting-javascript@master
```

According to the [documentation](https://my-wedolo.pm-projects.de/statistiken/standing_data) the branch is defined by the param `ref`

```suggestion
          repository: pmagentur/linting-javascript
          ref: master
```

---

pretty sure you send the wrong link ;) can you update it? 
[here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) It just gives a version as example:
```
with:
          repository: octocat/my-private-repo
          ref: v1.0
```